config/features.yml

```diff
@@ -121,6 +121,10 @@ features:
     actor_type: user
     description: Enables DOB instead of last 4 of SSN.
     enable_in_development: false
+  check_in_experience_edit_messaging_enabled:
+    actor_type: user
+    description: Enables showing of edit messaging.
+    enable_in_development: false
```

Comment:
Is there any reason to keep this feature disabled in development environment?

Reply:
@kanchanasuriya that's a good question. The ticket says to keep it off in dev, stage, and prod: https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/39399

Maybe because the work hasn't started yet?